pkg/manager/member/config_test.go

```diff
@@ -67,9 +70,10 @@ func TestUpdateConfigMap(t *testing.T) {
 
 	tests := []testcase{
 		{
-			name: "two empty config map",
-			old:  &corev1.ConfigMap{},
-			new:  &corev1.ConfigMap{},
+			name:  "two empty config map",
+			old:   &corev1.ConfigMap{},
```

Comment:
Please also add a case where both CMs have the same data.

Reply:
ok